.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Description of changes  
+Provide a description of what this PR does. What bug does it fix, or what feature does it add? Do you expect that this PR will change answers, and if so, under what circumstances?
+
+## Tests conducted
```

Comment:
Can we combine or move this Tests conducted section into or above the Regression tests section below?

Reply:
Since this section is for the developer to tell us what tests they conducted themselves (separate from the regression tests that will be conducted later), I think it is a good idea to keep this section. But I moved it to just above the regression tests section so that all of the tests will be together.